docs/federate.md

```diff
@@ -99,8 +126,8 @@ to launch the `f-API` together with a connected query tool.
     as described in the [official documentation](https://docs.docker.com/engine/install/).
 
 Copy the following snippet into your `docker-compose.yml` file.
-You should not have to change anything about this file.
-All local configuration changes are done in the `fed.env` file.
+**You should not have to change anything about this file.
+All local configuration changes should be made in either the `local_nb_nodes.json` or `fed.env` files.**
```

Comment:
:cherries: bold highlight looks a bit funky (see staging build). If you think this needs a separate visual highlight, maybe make a box? otherwise I think plaintext works